src/modules/dashboard/components/CreateEditDomainDialog/CreateEditDomainDialogForm.css

```diff
@@ -0,0 +1,20 @@
+.displayFlex {
```

Comment:
Again, this is a very bad class name. Please stop naming classes after css attributes.

Please come up with a descriptive name for this.